xml/SLES-JeOS-12SP4-Quickstart.xml

```diff
@@ -350,7 +350,7 @@
     version.</para>
 
    <para>For example, the hypervisors supported with SUSE Linux Enterprise Server 15 can be
-    found in the product documentation at &#8594; <link xlink:href="https://www.suse.com/documentation/sles-15/singlehtml/book_virt/book_virt.html#virt.support.hosts">https://www.suse.com/documentation/sles-15/singlehtml/book_virt/book_virt.html#virt.support.hosts</link></para>
+    found in the product documentation at &#8594; <link xlink:href="https://documentation.suse.com/sles/15-SP1/html/SLES-all/cha-virt-support.html">https://documentation.suse.com/sles/15-SP1/html/SLES-all/cha-virt-support.html</link></para>
```

Comment:
1. Please link to the correct section: https://documentation.suse.com/sles/15-SP1/html/SLES-all/cha-virt-support.html#virt-support-hosts (note the part after the hash!)

2. Using the generic .../sles-15/... link that redirects to the newest SLES 15 SP seems appropriate here.

3. Please remove the redundant link text while you are at it -- it will be generated automatically.

```suggestion
    found in the product documentation at &#8594; <link xlink:href="https://documentation.suse.com/sles-15/html/SLES-all/cha-virt-support.html#virt-support-hosts"/>.</para>
```

Reply:
Okay